.vscode/launch.json

```diff
@@ -0,0 +1,38 @@
+{
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Program",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "program": "${workspaceFolder}/dist/index.js",
```

Comment:
:robot: OpenAI

The `launch.json` file has an extra comma at the end of line 10, which may cause issues in some JSON parsers.
```suggestion
{
  "configurations": [
    {
      "type": "node",
      "request": "launch",
      "name": "Launch Program",
      "skipFiles": [
        "<node_internals>/**"
      ],
      "program": "${workspaceFolder}/dist/index.js"
    },
    {
      "type": "node",
      "request": "launch",
      "name": "Run Mocha tests",
      "program": "${workspaceRoot}/node_modules/mocha/bin/_mocha",
      "runtimeArgs": [
        "-r",
        "${workspaceRoot}/node_modules/source-map-support/register"
      ],
      "cwd": "${workspaceRoot}",
      "autoAttachChildProcesses": true,
      "args": [
        "--config",
        "${workspaceRoot}/.mocharc.json",
        "${workspaceRoot}/dist/__tests__/**/*.js",
        "-t",
        "0"
      ]
    },
    {
      "type": "node",
      "request": "attach",
      "name": "Attach to Process",
      "port": 5858
    }
  ]
}
```

<!-- This is an auto-generated comment by OpenAI -->